apps/api/src/app/workflows-v2/usecases/patch-workflow/patch-workflow.usecase.ts

```diff
@@ -11,18 +10,12 @@ export class PatchWorkflowUsecase {
   constructor(
     private getWorkflowByIdsUseCase: GetWorkflowByIdsUseCase,
     private notificationTemplateRepository: NotificationTemplateRepository,
-    private postProcessWorkflowUpdate: PostProcessWorkflowUpdate,
     private getWorkflowUseCase: GetWorkflowUseCase
   ) {}
 
   async execute(command: PatchWorkflowCommand): Promise<WorkflowResponseDto> {
     const persistedWorkflow = await this.fetchWorkflow(command);
-    let transientWorkflow = this.patchWorkflowFields(persistedWorkflow, command);
-
-    transientWorkflow = await this.postProcessWorkflowUpdate.execute({
-      workflow: transientWorkflow,
-      user: command.user,
-    });
+    const transientWorkflow = this.patchWorkflowFields(persistedWorkflow, command);
```

Comment:
what is the status of using the patch? do we want to keep it??
this change means that we won't update issues on patch.
if we need it i can recreate the issue generation on patch as well.

Reply:
For now, patch should only toggle the active status of a workflow. So you can safely remove everything else and simplify the PATCH usecase. See [here](https://www.notion.so/novuhq/Workflow-V2-API-159d03ad838f8065a49feaa44a7b3efe?pvs=4#159d03ad838f802d885df64ebd5b5869).

Reply:
amazing, thanks for the ref as well!